src/sentry/notifications/utils/participants.py

```diff
@@ -269,10 +269,19 @@ def get_release_committers(project: Project, event: Event) -> Sequence[User]:
 
     # commit_author_id : Author
     author_users: Mapping[str, Author] = get_users_for_commits(commits)
-    return list(
+    release_committers = list(
         User.objects.filter(id__in={au["id"] for au in author_users.values() if au.get("id")})
     )
 
+    return list(
+        filter(
+            lambda u: features.has(
```

Comment:
@barkbarkimashark Is there any plan to make a notification setting for this feature? I'm going to assume right now this is just to test to make sure this has some value at all before we worry about making it controllable.

Reply:
Yup you got it exactly right. This is all experimentation and validation at this point. I don't imagine this functionality looking like this 1-2 months from now.